spec/units/line_editor_blank_filter_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
```

Comment:
Credit where it's due: I just renamed this file, it was @sgrif and @adam89 who first wrote it.